src/styles/blocks/input-box.scss

```diff
@@ -0,0 +1,100 @@
+.input-box {
+  box-sizing: border-box;
+  height: 70px;
+  position: relative;
+  display: flex;
+  margin-top: 20px;
+
+  padding-left: 62px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba($color: #3d4e61, $alpha: 0.1);
+
+  &:has(&__input:hover) {
```

Comment:
you can simplify this selector by removing divs and delegating their role to the form or to the input